RtspServer/RtspServer.cs

```diff
@@ -279,14 +280,7 @@ internal IEnumerable<ClientSession> Clients
             [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
             get
             {
-                try
-                {
-                    return m_Sessions.Values.ToList();
-                }
-                catch
-                {
-                    throw;
-                }
+                return m_Sessions.Values.ToList();
```

Comment:
See Comment below, I think because your using Concurrent now you can skip the ToList.